samples/data-sealing/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.10)
+
+project("Data Sealing Sample" LANGUAGES C CXX)
+
+find_package(Threads REQUIRED)
+
+find_package(OpenEnclave CONFIG REQUIRED)
+add_executable(openenclave::oeedger8r IMPORTED)
+set_target_properties(openenclave::oeedger8r PROPERTIES IMPORTED_LOCATION ${OE_BINDIR}/oeedger8r)
+
+add_custom_command(OUTPUT ${CMAKE_SOURCE_DIR}/common/datasealing_t.c
```

Comment:
Why not just break this across the subdirectories, specifically with `common` having its own CMakeLists.txt? Could this just be a custom target that the enclave subdirectories depend on? It seems like something the Makefile implementation should have done as well, instead of having it as part of `enc1` specifically.

Reply:
See my description and commit for exactly why this happened this way 😉